src/xdocs/cmdline.xml.vm

```diff
@@ -143,7 +143,10 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           exclude from CheckStyle. Multiple excludes are allowed.
         </li>
         <li>
-          <code>-v</code> - print product version and exit. Any other option is ignored.
+          <code>-V, --version</code> - print product version and exit. Any other option is ignored.
```

Comment:
Most unix tools use `-V` (uppercase) for `--version` and `-v` (lowercase) for `--verbose`. This is not a standard but fairly common usage, as documented [here](http://catb.org/~esr/writings/taoup/html/ch10s05.html#id2948149).  Example tools that follow this convention: grep, curl, gzip, ssh.

Because of this background, picocli has a shortcut for providing `-h`, `--help` for usage help, and `-V`, `--version` for version help: these options are "mixed in" when the command is defined as `@Command(mixinStandardHelpOptions = true)`. I used this shortcut to avoid the need for defining annotated fields for these standard options.

Unfortunately the shortcut does not allow customizing the option names. If the name needs to be changed, we need to define an annotated field: `@Option(names = "-v") boolean isVersionRequested;`. I thought it better to use the `mixinStandardHelpOptions` shortcut.

Reply:
good to know.
We can "-V" I see no problem with it.